packages/log/data_stream/log/manifest.yml

```
@@ -17,8 +17,20 @@ streams:
         title: Dataset name
         description: >
           Set the name for your dataset. Changing the dataset will send the data to a different index. You can't use `-` in the name of a dataset and only valid characters for [Elasticsearch index names](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html).
-
         type: text
+      - name: tags
+        type: text
+        title: Tags
```

```suggestion
        title: Tags
        description: Tags to include in the published event
```

For consistency with other custom inputs